docs/en/04_Changelogs/4.1.0.md

```diff
@@ -18,7 +18,7 @@ projects (updating from 3.x or 4.0) will continue working as-is, but we
 strongly recommend switching to the public webroot structure in order to
 get the security benefits.
 
-This folder name is not configurable, but is turned on by creating this folder, and off by ensuring
+> This folder name is not configurable, but is turned on by creating this folder, and off by ensuring
```

Comment:
why add the leading `>`?

Reply:
Appearance's sake, this sentence needed to be a bit more obvious